ssts/segmentation/example.py

```diff
@@ -58,7 +58,7 @@ def main(data_path=None, example_number=None):
         mm.show_distributions_together(labels, no_outliers_data)  # all classes plotted together
         mm.show_distributions_separately(labels, no_outliers_data)  # each class plotted per plot
 
-    ## NOTE Segmentation example with dimensionality reduction (PCA) across physical properties
+    ## NOTE Segmentation example with dimensionality reduction (PCA) across physical properties AND grain segmentation
```

Comment:
Example 2 does PCA segmentation across properties (not using neighbor information) and does grain segmentation